nbgrader/api.py

```diff
@@ -111,6 +111,9 @@ class Notebook(Base):
     #: Unique id of :attr:`~nbgrader.api.Notebook.assignment`
     assignment_id = Column(String(32), ForeignKey('assignment.id'))
 
+    #: The json string representation of the kernelspec for this notebook
+    kernelspec = Column(String(128), nullable=True)
```

Comment:
I wonder if we potentially want to make this really long (e.g. 1024 characters) just in case the kernelspec is ever really long for some reason.